scripts/globals/abilities/call_wyvern.lua

```diff
@@ -33,4 +33,6 @@ end;
 
 function onUseAbility(player,target,ability)
     player:spawnPet(PET_WYVERN);
+    local pet = player:getPet();
+    pet:addMod(MOD_DMG, -40);
```

Comment:
I don't think this persist through zoning and needs to be "baked into" the object before you spawn it using mob_pool_mods.sql or else where the core builds our flying scaled friends stats in the first place. Possibly could put it in the global `scripts\pets\wyvern.lua` but I think thats intended to handle listeners only and dunno if we want to expand that to other things.

The pool ID for a players Wyvern is 5551.